### PR TITLE
Label and Dropdown Updates

### PR DESCRIPTION
**Pull request by philschanely**
## Description

This PR adds some new features to Label and SelectDropdown to ease composition for common patterns:

- Adds `LabelSecondaryButton` that encapsulates settings that need to be used consistently for the `secondaryButton` option in `Label`s.
- Adds a `LabelGroup` component for display inline/wrapping lists of `Label`s.
- Adds an option on `SelectDropdown` for showing selected items as `Label`s below the dropdown that can also be removed (React only)

### Screenshots

New `showSelectionsAsLabels` option on React SelectDropdown:

Screen Shot 2021-02-10 at 10 41 49 AM

No visual changes otherwise.

Bonus: Adds `fullWidth` prop to React Button to sync up with the one in Rails. 

## Test notes

See Elements > Label and Elements > Dropdown to verify no visual changes or errors.
See Storybook > Dropdown > Multiselect to see new component in use.

### Steps for testing

These are new features that do not create issues with existing use of these components but do open up opportunities to simplify some existing implementations.

Verify existing uses are unaffected in the following:

1. Label: ensure labels look same and function as they have to date:

    - [ ] Product outline editor (React): publish status dropdowns
    - [ ] Contact import bulk actions step (React): Tag selections that appear below dropdown
    - [ ] Settings > Domain (Rails): see `Kajabi default` is displaying as before
    - [ ] Offers (Rails): publish status label appears as expected (FYI: clicking off of this dropdown does not appear to function as expected and dismiss the dropdown. This appears to be the functionality before this PR as well. A patch may be needed to how this component is implemented here.)

1. Dropdown:
    - [ ] Product outline editor (React): any existing dropdowns
    - [ ] Contact import bulk actions step (React): Tag selection dropdown
    - [ ] Offers (Rails): ... options menu
    - [ ] Product catalogue (Rails): any dropdowns

BONUS: Button: no changes should affect React-based buttons. Confirm that any buttons in the follow appear as expected:
  - [ ] New Contact import process
  - [ ] Product outline manager

### CODE REVIEW

**Review by monkeypox8 — COMMENTED** [review]
Seeing some errors trying to navigate through the docs. Did I miss something?

elements
objects

**Comment by philschanely** [comment]
@monkeypox8 I've fixed this. Sorry for the trouble!

File: packages/sage-react/lib/Dropdown/SelectDropdown.jsx [hunk 1]
```diff
@@ -69,6 +71,7 @@ export const SelectDropdown = ({
   };
 
   const changeValue = (data) => {
+    console.log('change', data);
```
Comment (joedakroub) [inline packages/sage-react/lib/Dropdown/SelectDropdown.jsx:74]: Is this intentional?

**Review by joedakroub — APPROVED** [review]
This is looking good locally. I checked labels and dropdowns in Offers specifically and everything is looks correct. Added one minor comment. 👍🏼

File: docs/app/views/examples/elements/label/_props.html.erb [hunk 2]
```diff
@@ -1,12 +1,71 @@
...
+<tr>
+  <td><%= md("`attributes`") %></td>
+  <td><%= md('Additional attributes such as to provide `data-` bindings or actions.') %></td>
+  <td><%= md('`String`') %></td>
+  <td><%= md('`"Remove"`') %></td>
+</tr>
+<tr>
+  <td><%= md("`icon`") %></td>
+  <td><%= md('
+    Incidates which icon to use for the button.
```
Comment (monkeypox8) [inline docs/app/views/examples/elements/label/_props.html.erb:55]: Tiny typo here, "indicates" (line 64 as well)

**Review by monkeypox8 — APPROVED** [review]
LGTM, couple of small typos but nothing blocking! Love the multiselect interaction in Storybook, it's really slick.

File: docs/app/views/examples/elements/label/_props.html.erb (hunk 2 above)
Comment (philschanely) [reply docs/app/views/examples/elements/label/_props.html.erb:55]: Sorry missed this! Will earmark to fix with documentation updates forthcoming.

**Pull request merged**